<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QWidget" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Папка с рядами:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPath">
       <property name="text">
        <string>../set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browseSetPath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="refreshPlot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Обновить график</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="savePlot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сохранить график в файл…</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as">
         <normaloff/>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveAll">
       <property name="text">
        <string>Сохранить в файлы графики всех пар</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Команда, которая будет запущена для сборки pdf всех графиков в один файл «plots.pdf» по нажатию кнопки «Сохранить в файл графики всех пар».&lt;/p&gt;&lt;p&gt;«%1» будет заменено на названия всех файлов через пробел; «%2» будет заменено на имя выходного файла.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Команда для склейки pdf:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pdftk">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Команда, которая будет запущена для сборки pdf всех графиков в один файл «plots.pdf» по нажатию кнопки «Сохранить в файл графики всех пар».&lt;/p&gt;&lt;p&gt;«%1» будет заменено на названия всех файлов через пробел; «%2» будет заменено на имя выходного файла.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>pdftk %1 output '%2'</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progressMsg">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCustomPlot" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Вертикальная ось:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axisY"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Горизонтальная ось:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="axisX"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>progressBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressMsg</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressMsg</sender>
   <signal>windowIconTextChanged(QString)</signal>
   <receiver>progressBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
